src/app/model/group/player.py

```diff
@@ -1,8 +1,19 @@
+from dataclasses import dataclass
+
 from src.app.model.base import Base
 
 
+@dataclass
 class Player(Base):
     """ Tracked player information """
-    name = ''
-    streak = 0
-    score = 0
+    name: str
+    streak: int
+    score: int
+
+    @classmethod
+    def from_dict(cls, dic):
+        return cls(
+            name=dic["name"],
```

Comment:
So we are still setting the lowest object values as a dict? Is this not something we were trying to remove?

Reply:
We are getting away from the dicts.

redis (json) -> redis client (dict) -> base class (python object)